plugins/fr.cea.nabla.ir/src/fr/cea/nabla/ir/generator/dace/DaceGenerator.xtend

```diff
@@ -41,12 +41,12 @@ class DaceGenerator implements IrCodeGenerator
 		UnzipHelper::unzipNRepository(wsPath)
 	}
 
-	override getName() { "Python" }
+	override getName() { "DaCe" }
 
 	override getIrTransformationSteps()
 	{
 		#[
-			new ReplaceReductions(true), // parallel loops and reductions not yet implemented
+			new DaceReplaceReductions(true), // parallel loops and reductions not yet implemented
```

Comment:
Target-specific IR Transformations should be handled differently. The ReplaceReductions transformation is already called as part of general IR transformations. Thus, modifying this line has no effect.

Reply:
Commenting this line _does_ break code generation.